tests/ci/build.yaml

```diff
@@ -61,9 +61,27 @@ steps:
   name: 'gcr.io/redpandaci/builder'
   args: ['./build/venv/v/bin/vtools', 'test', 'go', '--conf=vtools/vtools/artifacts/ci/vtools-gcc-release.yml']
 
+- id: 'restore ccache'
+  name: 'docker:20.10.3'
+  args: ['run', '--ipc=host', '--network=cloudbuild', 'gcr.io/redpandaci/restore_cache',
```

Comment:
Is there some reason/issue for using docker in docker in these steps?

Reply:
yeah, the `--ipc=host` flag is needed in order to access the host's `/dev/shm` temp filesystem. and this is not exposed on a GCB's steps options